scrapers/sis_scraper/main.py

```diff
@@ -90,7 +91,7 @@ async def get_section_information(section_url):
     return section_dict
 
 
-async def get_class_information(class_url):
+async def get_class_information(class_url, registration_dates):
```

Comment:
Why is this using an out-param instead of returning two values?

Reply:
Changed in latest commit to use multiple return values.